torch/_inductor/mkldnn.py

```diff
@@ -295,9 +295,9 @@ def pack_module(gm: torch.fx.GraphModule):
                     continue
                 if dynamo_config.dynamic_shapes:
                     computation_node_input_size = None
-                    if (
+                    if not (
                         type(cur_module) in [torch.nn.Linear]
-                        and cur_module.weight.dtype == torch.float32
+                        and cur_module.weight.dtype == torch.bfloat16
```

Comment:
Suggest to add comment on why we only enable prepacking for bfloat16 linear on dynamic shapes.

Reply:
added